components/entitlement/org.wso2.carbon.identity.entitlement/src/test/resources/dbscripts/h2.sql

```diff
@@ -0,0 +1,1570 @@
+    UNIQUE (MAIN_APP_ID, OWNER_ORG_ID, SHARED_ORG_ID),
+    UNIQUE (SHARED_APP_ID)
+);
+
+CREATE TABLE IF NOT EXISTS IDVP (
+	ID          INTEGER NOT NULL AUTO_INCREMENT,
+	UUID        CHAR(36) NOT NULL,
+	TENANT_ID   INTEGER NOT NULL,
+	IDVP_TYPE   VARCHAR(254),
+	NAME        VARCHAR(254),
+	DESCRIPTION VARCHAR(1024),
+	IS_ENABLED  CHAR(1) NOT NULL DEFAULT '0',
+	PRIMARY KEY (ID),
+	UNIQUE (TENANT_ID, NAME),
+    UNIQUE (UUID)
+);
+
+CREATE TABLE IF NOT EXISTS IDVP_CLAIM_MAPPING (
+    ID            INTEGER NOT NULL AUTO_INCREMENT,
+    IDVP_ID       INTEGER NOT NULL,
+    TENANT_ID     INTEGER NOT NULL,
+    CLAIM         VARCHAR(254),
+    LOCAL_CLAIM   VARCHAR(254),
+    PRIMARY KEY (ID),
+    UNIQUE (IDVP_ID, CLAIM, TENANT_ID),
+    FOREIGN KEY (IDVP_ID) REFERENCES IDVP(ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDVP_CONFIG (
+	ID              INTEGER NOT NULL AUTO_INCREMENT,
+	IDVP_ID         INTEGER NOT NULL,
+	TENANT_ID       INTEGER NOT NULL,
+	PROPERTY_KEY    VARCHAR(254) NOT NULL,
+  	PROPERTY_VALUE  VARCHAR(1024),
+  	IS_SECRET       CHAR (1) DEFAULT '0',
+	PRIMARY KEY (ID),
+	UNIQUE (IDVP_ID, PROPERTY_KEY, TENANT_ID),
+	FOREIGN KEY (IDVP_ID) REFERENCES IDVP(ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDV_CLAIM (
+	ID          INTEGER NOT NULL AUTO_INCREMENT,
+	UUID        CHAR(36) NOT NULL,
+	USER_ID     VARCHAR(254) NOT NULL,
+	CLAIM_URI   VARCHAR(254),
+	IDVP_ID     CHAR(36) NOT NULL,
+	TENANT_ID   INTEGER NOT NULL,
+	IS_VERIFIED CHAR(1) NOT NULL DEFAULT '0',
+	METADATA    BLOB,
+	PRIMARY KEY (ID),
+	UNIQUE (CLAIM_URI, TENANT_ID, USER_ID, IDVP_ID),
+    UNIQUE (UUID),
+    FOREIGN KEY (IDVP_ID) REFERENCES IDVP(UUID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH_PAR (
+    REQ_URI_REF VARCHAR(255) PRIMARY KEY,
+    CLIENT_ID VARCHAR(255) NOT NULL,
+    SCHEDULED_EXPIRY BIGINT NOT NULL,
+    PARAMETERS MEDIUMTEXT
+);
+
+CREATE TABLE IF NOT EXISTS IDN_ORG_USER_INVITATION (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    INVITATION_ID VARCHAR(40) NOT NULL,
+    CONFIRMATION_CODE VARCHAR(40) NOT NULL,
+    USER_NAME VARCHAR(254) NOT NULL,
+    DOMAIN_NAME VARCHAR(254) NOT NULL,
+    EMAIL VARCHAR(254) NOT NULL,
+    USER_ORG_ID VARCHAR(254) NOT NULL,
+    INVITED_ORG_ID VARCHAR(254) NOT NULL,
+    USER_REDIRECT_URL VARCHAR(1024) NOT NULL,
+    STATUS VARCHAR(10) NOT NULL,
+    CREATED_AT TIMESTAMP NOT NULL,
+    EXPIRED_AT TIMESTAMP NOT NULL,
+    PRIMARY KEY (INVITATION_ID)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_ORG_USER_INVITE_ASSIGNMENT(
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    INVITATION_ID VARCHAR(40) NOT NULL,
+    ASSIGNMENT_ID VARCHAR(255) NOT NULL,
+    ASSIGNMENT_TYPE VARCHAR(255) NOT NULL,
+    PRIMARY KEY (INVITATION_ID, ASSIGNMENT_ID, ASSIGNMENT_TYPE),
+    FOREIGN KEY (INVITATION_ID) REFERENCES IDN_ORG_USER_INVITATION(INVITATION_ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS API_RESOURCE (
+    ID CHAR(36) NOT NULL PRIMARY KEY,
```

Comment:
No primary key ?

Reply:
This is the same script we have in the latest IS snaspshot build and here I used the same script to initialized in memory h2 database for unit test cases.